lib/dirList.js

```diff
@@ -149,7 +149,7 @@ const dirList = {
       route = path.normalize(path.join(route, '..'))
     }
     return {
-      href: path.join(prefix, route, entry.name).replace(/\\/gu, '/'),
+      href: encodeURI(path.join(prefix, route, entry.name).replace(/\\/gu, '/')),
```

Comment:
I wonder, can we avoid this regex?

Reply:
This regex was already there. I tried switching to `path.posix.join` but that failed the tests on Windows.